dbms/src/Functions/tests/gtest_arithmetic_functions.cpp

```diff
@@ -103,6 +103,29 @@ class TestBinaryArithmeticFunctions : public DB::tests::FunctionTest
     }
 };
 
+TEST_F(TestBinaryArithmeticFunctions, TiDBDivideDecimalRound)
```

Comment:
And add test cases for Decimal32, Decimal128 and Decimal256?

Reply:
added